src/PrestaShopBundle/Form/Admin/Product/ProductInformation.php

```diff
@@ -220,12 +220,8 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'type' => FormType\TextType::class,
                 'options' => [
                     'constraints' => [
-                        new Assert\Regex([
-                            'pattern' => '/[<>;=#{}]/',
-                            'match' => false,
-                        ]),
                         new Assert\NotBlank(),
-                        new Assert\Length(['min' => 3, 'max' => 128]),
+                        new Assert\Length(['min' => 1, 'max' => 128]),
```

Comment:
may be you can re-use here the public constant you create previously if you apply my comment.